templates/Collector/mapping_collector.html.twig

```diff
@@ -20,7 +20,7 @@
     {# This left-hand menu appears when using the full-screen profiler. #}
     <span class="label">
         <span class="icon">
-            {% include '@VichUploader/Collector/icon_white.html.twig' %}
+            {{ source('@VichUploader/Collector/icon.svg') }}
```

Comment:
I didn't remove anything because I didn't want to be 'destructive'. I figured you could remove what you don't want.

I don't see where you use different icons based on theme.

the PR is simply a proposal to match the way it is done in Symfony and explained [here](https://symfony.com/doc/current/profiler.html#adding-web-profiler-templates)

Reply:
> So you are proposing the same icon?

it isn't the same icon. It is a _similar_ icon, but more in the current style of the toolbar

Reply:
I mean a single icon for light/dark.
If understand correctly, you took the icon from tabler.io, which seems to support both.

Could you provide screenshots of the new icon in both themes?

Reply:
The Symfony profiler has a "settings" link in the lower right corner.

Just like Symfony does, we need to add a license file. See https://github.com/symfony/symfony/blob/7.0/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Icon/LICENSE.txt to see how Symfony is doing it

Reply:
<img width="258" alt="Screenshot 2023-12-06 at 10 01 08 AM" src="https://github.com/dustin10/VichUploaderBundle/assets/350048/bb273604-8dd1-4d1d-ba0e-cb4b8e7602e3">
<img width="64" alt="Screenshot 2023-12-06 at 10 01 27 AM" src="https://github.com/dustin10/VichUploaderBundle/assets/350048/737997c6-131d-4358-b2b6-32e18b9d09b9">
<img width="251" alt="Screenshot 2023-12-06 at 10 01 51 AM" src="https://github.com/dustin10/VichUploaderBundle/assets/350048/b8fa3c89-8973-417e-aa87-30d253311217">
<img width="260" alt="Screenshot 2023-12-06 at 10 01 14 AM" src="https://github.com/dustin10/VichUploaderBundle/assets/350048/dd883b86-76a4-4a81-aaf9-8fbc6b206ea0">
<img width="264" alt="Screenshot 2023-12-06 at 10 02 02 AM" src="https://github.com/dustin10/VichUploaderBundle/assets/350048/9a80cfb1-e8bc-467c-a317-4f61266a89ea">

Reply:
OK, so let's go with this icon, as soon as:
* old icons are removed
* a license file is added in the same dir

Reply:
✅